_tour/basics.md

```diff
@@ -266,17 +265,17 @@ We will cover objects in depth [later](singleton-objects.html).
 
 ## Traits
 
-Traits are types containing certain fields and methods.  Multiple traits can be combined.
+Traits are abstract data types containing certain fields and methods. In Scala inheritance, a class can not only inherit from a super class, but also mix in one or more traits.
```

Comment:
This is phrased somewhat awkwardly, though the point is good.  Maybe something like, "A class can only extend a single other class, but it can extend multiple traits."  (Caveat: I'm not sure "extend" is the right word to use.)

Reply:
I don't know whether extend or mix in is more accurate.

In the [Traits](https://docs.scala-lang.org/tour/traits.html) page it says "Classes and objects can **extend** traits", and in the [mixins](https://docs.scala-lang.org/tour/mixin-class-composition.html) page it says "**Mixins** are traits which are used to compose a class.", so do you mix in a mixin, or do you extend a mixin?